emscripten.py

```diff
@@ -312,19 +310,26 @@ def update_settings_glue(settings, metadata):
   # Integrate info from backend
   if settings['SIDE_MODULE']:
```

Comment:
Cool, went with 3 in https://github.com/kripken/emscripten/pull/5160/commits/438b29117ce6259919216052be3c3c25abd8dcf1